src/main/java/eu/europeana/metis/sandbox/config/amqp/QueueConsumerConfig.java

```diff
@@ -5,22 +5,22 @@
 import org.springframework.amqp.support.converter.MessageConverter;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.amqp.SimpleRabbitListenerContainerFactoryConfigurer;
+import org.springframework.context.annotation.Configuration;
 
 /**
  * The type Queue consumer configuration for workflow listeners.
  * Every listener has a {@link SimpleRabbitListenerContainerFactory}.
  * <br /> If changes like increasing consumers for a listener are needed,
- * every subclass can be customized by using the SimpleRabbitListenerContainerFactory
- * by default 2 consumers, 2 concurrent and 1 prefetch is set if none is defined
- * subclasses may also define their own defaults.
+ * every subclass can be customized by using the SimpleRabbitListenerContainerFactory.
  */
-public abstract class QueueConsumerConfiguration {
+@Configuration
+public abstract class QueueConsumerConfig {
 
-  @Value("${spring.rabbitmq.listener.simple.concurrency:2}")
```

Comment:
the original name of the property is here.
https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html
that is why it is named 
```
spring.rabbitmq.listener.simple.concurrency
spring.rabbitmq.listener.simple.max-concurrency
```

Reply:
After having this review, it brought to the surface the two different names of properties. Do you agree if we standardise into one name and use this spring value names? 
```
.concurency
.max-concurrency
.prefetch
```
with default values
```
spring.rabbitmq.listener.simple.concurrency:2
spring.rabbitmq.listener.simple.max-concurrency:2
spring.rabbitmq.listener.simple.prefetch:1
```